#!/usr/bin/env python
PACKAGE = "modelos_cinematicos"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("Angle_qy",    double_t,    0, "Orientaciòn de qy deseada 0 a 90 (Cuadrante I) / 90 a 180 (Cuadrante II)", 10,  0,180)
gen.add("Grafica_Python", bool_t , 0, "Dibuja la gràfica de Espacio de trabajo en Python",False)
gen.add("Workspace", bool_t , 0, "Dibujar espacio de trabajo",False)
gen.add("Borrar_workspace", bool_t , 0, "Borrar espacio de trabajo",False)
gen.add("Cuadrado", bool_t , 0, "Realizar Trayectoria cuadrada",False)
gen.add("Triangulo", bool_t , 0, "Realizar Trayectoria triangular",False)
gen.add("Circulo", bool_t , 0, "Realizar Trayectoria Circular",False)
gen.add("Simulacion", bool_t , 0, "Realizar trayectoria unicamente en simulacion",False)
gen.add("Dynamixel", bool_t , 0, "Realizar trayectoria con robot reay simulado",False)
gen.add("Solo_Dynamixel", bool_t , 0, "Realizar trayectoria unicamente con robot real",False)
gen.add("Ready", bool_t , 0, "Marcar para iniciar trazo de trayectoria",False)
exit(gen.generate(PACKAGE, "trayectorie_3dof", "figuras_3g"))
